src/ILLink.Tasks/build/Microsoft.NET.ILLink.targets

```
@@ -212,6 +221,10 @@ Copyright (c) .NET Foundation. All rights reserved.
          potentially problematic when warnings are suppressed. -->
     <NETSdkInformation Condition="'$(PublishTrimmed)' == 'true' And '$(SuppressTrimAnalysisWarnings)' == 'true'" ResourceName="ILLinkOptimizedAssemblies" />
 
+    <!-- In .NET 7, TrimmerDefaultAction is deprecated. TrimMode can be used for the supported configurations. -->
+    <Warning Condition="'$(TrimmerDefaultAction)' != '' And $([MSBuild]::VersionGreaterThan('$(TargetFrameworkVersion)', '6.0'))"
+             Text="Property 'TrimmerDefaultAction' is deprecated in .NET 7." />
```

```suggestion
             Text="Property 'TrimmerDefaultAction' is deprecated in .NET 7 and will be ignored." />
```

Could even go further and be a bit more descriptive on what to do.